docs/usage.rst

```diff
@@ -173,10 +192,11 @@ read the mask. Typically, OpenMP will bind threads to cores if the ``OMP_PROC_BI
 binding of the main thread only at the first parallel region which means that if pika is initialized
 before the first parallel region, the mask will most likely be read correctly. Other implementations
 (e.g. GNU) set the binding of the main thread in global constructors which may run before pika can
-read the process mask. In that case you may need to either use ``--pika:ignore-process-mask`` to use
-all cores on the system or explicitly set a mask with ``--pika:process-mask``. If there is a process
-mask already set in the environment that is launching the application (e.g. in a SLURM job) you can
-read the mask before the application runs with hwloc:
+read the process mask. In that case you may need to either use
+``PIKA_IGNORE_PROCESS_MASK``\/``--pika:ignore-process-mask`` to use all cores on the system or
+explicitly set a mask with ``--pika:process-mask``. If there is a process mask already set in the
```

Comment:
Probably overkill, but would it be worth linking to the section below already here? In case one stops reading after the code block and misses the `pika-bind` script?

Reply:
I don't think it hurts! I for sure recognize myself stopping reading at some arbitrary point and not getting to the `pika-bind` section...

Reply:
I added a link right before the example: https://github.com/pika-org/pika/pull/1387/commits/4cc50628dfc4b9b62829ea7806ebe2537e1ae0bf.